README.md

```diff
@@ -57,7 +60,7 @@ var HUMLValidator = function() {
     }
 }
 ```
-As this is a validation rule, we will throw an error `your age HAS to be greater than 22` if the criteria specified in the rule is not met.
+As this is a validation rule, we will throw an error "`your age HAS to be greater than 22`" if the criteria specified in the rule is not met.
```

Comment:
Why the qutation marks if it's already in a code block?

Reply:
I'm quoting the error message.